db/schema.rb

```diff
@@ -57,7 +56,6 @@
     t.bigint "reviewer_id"
     t.datetime "reviewed_at", precision: nil
     t.string "pr_state"
-    t.index ["company_id"], name: "index_contributions_on_company_id"
     t.index ["link"], name: "index_contributions_on_link", unique: true
```

Comment:
Should these other columns in unrelated tables be deleted from the schema?

Perhaps you need to run your migration on top of the existing schema (the one on `master`), but this is more of a guess.